src/components/CatalogOverlay/CatalogPlotComponent/CatalogPlotComponent.tsx

```diff
@@ -880,11 +880,11 @@ export class CatalogPlotComponent extends React.Component<WidgetProps> {
             layout.xaxis.range = [border.xMin, border.xMax];
             layout.yaxis.range = [border.yMin, border.yMax];
             layout.yaxis.title = widgetStore.yColumnName;
-            layout.yaxis.tickformat = this.formatTickValues(layout.yaxis.range);
             layout.xaxis.tickformat = this.formatTickValues(layout.xaxis.range);
```

Comment:
I prefer we use `[border.xMin, border.xMax]`.

Reply:
I modified them.